deploy/dev-env.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash -e
+
+usage() {
+  cat <<EOF
+Usage: $0 [option]
+
+options:
+  -b, --rebuild              Build new image and push it to your local registry
+                               before cluster creation
+  -c, --recreate             Create a new cluster based on the last build image
+
+For debugging run "bash -x $0 [option]"
+EOF
+}
+
+exists(){
+  command -v "$1" >/dev/null 2>&1
+}
+
+step1_run_local_registry(){
+  registry_is_running=$(podman ps | grep "registry:2")
```

Comment:
If the registry is not running the script aborts because of `-e` in the shebang.
```
$ podman ps | grep "registry:2"
$ echo $?
1
```
Additionally, all `podman` commands must be run with `sudo` because all other commands like `run-registry` or `build-containers` are doing the same which would otherwise result in 
```
Writing manifest to image destination
Storing signatures
cddb6982775b8605519b7c1c93b8fb1b888b38d53105c27c5d9b064bda6b8e47
6f1ab5a3b7087a6e41719123e2498ef2329e9cc8cc15cb281e4d8ea2d9a122f9
Error: localhost/opensuse/bubbles:master: image not known
```
```suggestion
  registry_is_running=$(sudo podman ps | grep "registry:2" || :)
```